src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -21,21 +23,28 @@
 
     private WebDriver driver;
     private static final int DELAY = 20;
+    private String url = System.getProperty("url");
 
+    /**
+     * Constructor creates new driver.
+     */
     public ChplDownloadSteps() {
         driver = Hooks.getDriver();
+        if (StringUtils.isEmpty(url)) {
+            url = "http://localhost:3000/";
+        }
     }
 
     @Given("^user is on CHPL download page$")
     public void userIsOnCHPLHome() throws Throwable {
-        driver.get("https://chpl.ahrqdev.org/#/resources/download");
+        driver.get(url + "#/resources/download");
         WebDriverWait wait = new WebDriverWait(driver, DELAY);
         assertTrue(driver.getTitle().contains("Download the CHPL"));
     }
 
     @When("^user is not logged in$")
     public void userIsNotLoggedIn() throws Throwable {
-        driver.get("https://chpl.ahrqdev.org/#/resources/download");
+        driver.get(url + "#/resources/download");
```

Comment:
User is not logged in shouldn't navigate anywhere. For that matter, I think it might be better to just take out this login stuff entirely until we can figure out a way to actually do it